### PR TITLE
Test Boston Results Dashboard

### PR DESCRIPTION
**Pull request by kerrjatwit**
Adding results via Power BI to the Test Boston site.

### CODE REVIEW

File: ddp-workspace/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "@ngx-translate/core": "13.0.0",
     "@ngx-translate/http-loader": "6.0.0",
     "auth0-js": "9.14.0",
-    "auth0-lock": "11.27.0",
```
Comment (zyme) [inline ddp-workspace/package.json]: Are these changes required for the dashboard to render properly?  If not, please remove them--we try to avoid auto-upgrade (`^`) in our package.json.

File: ddp-workspace/package-lock.json [hunk 2]
```diff
@@ -5014,9 +5153,9 @@
       "dev": true
     },
     "dns-packet": {
-      "version": "1.3.1",
-      "resolved": "https://registry.npmjs.org/dns-packet/-/dns-packet-1.3.1.tgz",
-      "integrity": "sha512-0UxfQkMhYAUaZI+xrNZOz/as5KgDU0M/fQ9b6SpkyLbk3GEswDi6PADJVaYJradtRVsRIlF1zLyOodbcTCDzUg==",
+      "version": "1.3.4",
```
Comment (zyme) [inline ddp-workspace/package-lock.json]: Similar comment to the `package.json` changes.  If these changes aren't required, please revert to the original `package-lock.json`.

File: ddp-workspace/projects/ddp-testboston/src/app/components/app/app.component.html [hunk 3]
```diff
@@ -2,4 +2,4 @@
     <app-header></app-header>
     <router-outlet></router-outlet>
     <app-footer></app-footer>
-</div>
```
Comment (zyme) [inline ddp-workspace/projects/ddp-testboston/src/app/components/app/app.component.html]: If the only thing changing here is whitespace or newlines, please revert.

**Comment by zyme** [comment]
Overall looks good, thanks @kerrjatwit!  I have a few comments inline.  Could you please remove the `.vs` directory?  @MocanaAtBroad @aleksey-kuznetsov please take a look as well.  Maybe we should update `.gitignore` to to include `.vs` as well?

File: ddp-workspace/package.json (hunk 1 above)
Comment (MocanaAtBroad) [reply ddp-workspace/package.json]: Unless code requires, let's not upgrade the library.

**Comment by aleksey-kuznetsov** [comment]
Looks good to me except for `.vs` directory being included in the commit. Let's add it to the `.gitignore`.
I couldn't open the app itself but I ran the test suite and the dashboard looks good!

File: ddp-workspace/angular.json [hunk 4]
```diff
@@ -2591,5 +2591,8 @@
     "@schematics/angular:component": {
       "style": "scss"
     }
+  },
+  "cli": {
+    "analytics": "aec2ec9e-dcea-4c9b-8eee-f19198de5878"
```
Comment (MocanaAtBroad) [inline ddp-workspace/angular.json]: Google doesn't need to track our usage 😉

**Review by MocanaAtBroad — COMMENTED** [review]
Couple of very little things, but looks good! 
Just check that the BI iframe results look to your satisfaction on a mobile display.

**Review by MocanaAtBroad — APPROVED** [review]
👍
